Content.Server/Light/EntitySystems/ExpendableLightSystem.cs

```diff
@@ -137,6 +138,7 @@ private void UpdateVisualizer(Entity<ExpendableLightComponent> ent, AppearanceCo
                     _appearance.SetData(ent, ExpendableLightVisuals.Behavior, string.Empty, appearance);
                     var ignite = new IgnitionEvent(false);
                     RaiseLocalEvent(ent, ref ignite);
+                    EnsureComp<TimedDespawnComponent>(ent);
```

Comment:
This needs to be under an event handler for a new component and IgnitionEvent checking for Ignite false